api/src/opentrons/protocol_engine/commands/load_liquid.py

```diff
@@ -44,11 +48,16 @@ class LoadLiquidResult(BaseModel):
 class LoadLiquidImplementation(AbstractCommandImpl[LoadLiquidParams, LoadLiquidResult]):
     """Load liquid command implementation."""
 
-    def __init__(self, **kwargs: object) -> None:
-        pass
+    def __init__(self, state_view: StateView, **kwargs: object) -> None:
+        self._state_view = state_view
 
     async def execute(self, params: LoadLiquidParams) -> LoadLiquidResult:
         """Load data necessary for a liquid."""
+        liquids = self._state_view.liquid.get_all()
+
+        if not any(x.id == params.liquidId for x in liquids):
+            raise LiquidNotFoundError()
```

Comment:
We can simplify this a little bit with the `in` operator:

```suggestion
        if params.liquidId not in self._state_view.liquid.get_all():
            raise LiquidNotFoundError()
```

Edit: Oops, I misread this line. We can't do this exactly because we're looking at `x.id`, not `x`.